rust/rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
 # Keep synced with `Dockerfile`
```

Comment:
`Dockerfile` should update to 1.82 as well

Reply:
Clearly haven't had my coffee yet.